internal/service/organization/resource_organization_test.go

```diff
@@ -37,17 +40,92 @@ func TestAccConfigRSOrganization_Basic(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckMongoDBAtlasOrganizationExists(resourceName),
 					resource.TestCheckResourceAttrSet(resourceName, "org_id"),
-					resource.TestCheckResourceAttrSet(resourceName, "description"),
 					resource.TestCheckResourceAttr(resourceName, "description", description),
+					resource.TestCheckResourceAttr(resourceName, "api_access_list_required", "false"),
+					resource.TestCheckResourceAttr(resourceName, "restrict_employee_access", "false"),
+					resource.TestCheckResourceAttr(resourceName, "multi_factor_auth_required", "false"),
 				),
 			},
 			{
-				Config: testAccMongoDBAtlasOrganizationConfigBasic(orgOwnerID, "org-name-updated", description, roleName),
+				Config: testAccMongoDBAtlasOrganizationConfigBasic(orgOwnerID, updatedName, description, roleName),
 				Check: resource.ComposeTestCheckFunc(
 					testAccCheckMongoDBAtlasOrganizationExists(resourceName),
 					resource.TestCheckResourceAttrSet(resourceName, "org_id"),
-					resource.TestCheckResourceAttrSet(resourceName, "description"),
 					resource.TestCheckResourceAttr(resourceName, "description", description),
+					resource.TestCheckResourceAttr(resourceName, "api_access_list_required", "false"),
+					resource.TestCheckResourceAttr(resourceName, "restrict_employee_access", "false"),
+					resource.TestCheckResourceAttr(resourceName, "multi_factor_auth_required", "false"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccConfigRSOrganization_BasicAccess(t *testing.T) {
+	acc.SkipTestForCI(t)
```

Comment:
is it hard to run it in CI / GH?

Reply:
sorry this was missed, this requires setting a up a paying organization in cloud-dev that new organizations can be linked to, I couldn't get it to work with my personal organization in cloud-dev (even though I have billing enabled) so requires looking into setting up payments correctly